sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -9,7 +9,7 @@ export class AbortController {
     constructor(parentSignals?: AbortSignalLike[]);
     constructor(...parentSignals: AbortSignalLike[]);
     abort(): void;
-    readonly signal: AbortSignal;
+    get signal(): AbortSignal;
```

Comment:
We typically downlevel TypeScript `get` accessor signatures to `readonly` properties. Was that something we were doing on the older types file that didn't get carried in?

Reply:
I didn't realize this was an accessor before, yes we should use `downlevel-dts` here if so

Reply:
@willmtemple and @xirzec 
So to be clear, we should not ship a `get` accessor at all? if so, why?
If not, would https://github.com/Azure/azure-sdk-for-js/issues/11319 take care of this?

Reply:
I prefer to avoid them since:

1. They have TS compatibility concerns before/after 3.7
1. They are harder for developers to reason about, as property accesses are usually assumed to be cheap and without side-effects.

There are times where for compat purposes we can't avoid them (and maybe this is one of those times) but for new code I'd like to steer away from using getters.

Reply:
I turned off `extract-api` again. We already have an issue opened for it: https://github.com/Azure/azure-sdk-for-js/issues/10320.